eng/pipelines/pipeline-generation.yml

```diff
@@ -26,6 +26,8 @@ parameters:
     default: 76
   - name: APIReview_AutoCreate_Configurations
     default: 93
+  - name: Cache_Secrets_for_CPP
```

Comment:
What is the secret for? Are we not using DevOps caching any longer?

Reply:
The intent is to move away from Azure DevOps caching -- https://github.com/Azure/azure-sdk-for-cpp/pull/3329